desktop/main.js

```diff
@@ -120,7 +149,19 @@ const mainWindow = (() => {
             });
 
             app.on('before-quit', () => quitting = true);
-            app.on('activate', () => browserWindow.show());
+
+            let firstActivate = true;
```

Comment:
both this and the next block could benefit from a few comments explaining what they're doing 👍